modules/layout.styl

```diff
@@ -1,3 +1,8 @@
+// main
+
+#mainContent
```

Comment:
This seems like a donor site specific declaration. If so should be in that codebase. Also should be kebab case as a CSS declaration not camelcase.

Reply:
Moved into donation site and changed to kebab